feat: [CDS-70318]: GitOps at Org level (#2668)
harness/developer-hub: docs/continuous-delivery/gitops/harness-cd-git-ops-quickstart.md

  	+ Permission to apply CustomResourceDefinition.  
  	For more information, see [User-Facing Roles](https://kubernetes.io/docs/reference/access-authn-authz/rbac/#user-facing-roles) from Kubernetes.

## Note

> GitOps entities can be created at all three levels: account, organization, and project. The agent serves as the prerequisite for creating other entities. A GitOps agent associated with an account has the authority to create and control entities at all levels. On the other hand, an organization-level agent can manage entities within the organization and its projects, while a project-level agent is limited to managing entities at the project level.

Reviewer: ```suggestion
GitOps entities can be created at account, organization, and project levels. The GitOps Agent is needed to create other entities. A GitOps Agent associated with an account has the authority to create and control entities at all levels. However, an organization-level Agent can manage entities within the organization and its projects and a project-level Agent can only manage entities at a project level.
```
